content/en/account_management/rbac/log_management.md

```diff
@@ -13,61 +13,115 @@ further_reading:
 Ask your sales representative or customer success manager to enable this feature.
 </div>
 
+## Overview
+
+By default, Datadog offers 3 roles to manage user access: the Datadog Admin, Standard, and Read Only Roles. These roles are intentionally permissive so that your Datadog account can help improve organizational transparency, cooperation, and speed. But log data can sometimes contain sensitive information, and the wrong modification of log pipelines or indexes can have significant impact. For this reason, you can use Role Based Access Control to limit which users can read which log indexes, and which users can manage log-related account assets like pipelines, indexes, archives, etc.
+
+
 ## Roles
 
-Within Log Management, you can specify which users can read which pieces of log data, and which users can manage log-related account assets, like pipelines, indexes, archives, etc.
+A user can only read on a log index or modify a log-related asset if they belong to one or more roles that grant permissions to read on the log index or modify the log-related asset. When a user belongs to more than one role, their permissions are cumulative: they are able to act by any of those roles' permissions. Permissions only grant access, they never reduce access. 
 
-The [out of the box roles][1] affect log management in the following way:
+For example, if some of your organization's logs are sensitive, you might grant the default roles read access on only the `main` index, and no write access on any processing pipelines. And then you might create custom roles for each team or business unit that grant them read access to only those indexes that contain the sensitive log events, and that grant each team write access only on those processing pipelines that impact their own logs.
+
+
+## Permissions
+
+Permissions grant the ability to read data or manage account assets in Datadog. By belonging to roles that have permissions, users are able to read data and configure assets in Datadog.
+
+Some permissions can be granted over a limited scope. If a role grants a permission over a limited scope, then that role only grants to its users that permission within its scope. For example, one organization may grant a widely available role to have read permissions on logs (logs_read_index_data), but only within the scope of the "indexes" called "main" and "http" while reserving more sensitive indexes for less widely available roles. If a permission is granted without any "scope" argument, it is a permission not limited by a scope. So if this logs_read_index_data permission were granted to some role without any scope, users of that role would be granted read access on all log indexes. 
```

Comment:
make explicit here which permissions are scoped, and what are they scoped against (index, pipelines). Let's mention here that a scoped permission can be set in the UI (and show caption along with the corresponding permission hereafter), or through API.